pipelines/nextflow/modules/genome_metadata/dump_genome_stats.nf

```diff
@@ -0,0 +1,40 @@
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+process DUMP_GENOME_STATS {
+    tag "${db.species}"
+    label "normal"
+    maxForks 10
+
+    input:
+        val server
+        val db
+
+    output:
+        tuple val(db), path("core_stats.json")
+
+    script:
+        def output = "core_stats.json"
+        """
+        touch $output
+        genome_stats_dumper --host '${server.host}' \
+            --port '${server.port}' \
+            --user '${server.user}' \
+            --password '${server.password}' \
```

Comment:
Something for the hackathon: we could create a function that rolls out a `server` into this set of arguments to pass along (seems something we are going to use quite frequently, and could adapt to when there is password and when there is not).

Reply:
Yes this is easy to provide as a groovy utils function